policyengine/countries/uk/reform_parameters.yaml

```diff
@@ -23,6 +23,30 @@ reforms:
         unit: currency-GBP
         tests:
           - decreases_net_income: true
+  wealth_tax:
+    rate:
+      description: A yearly flat tax on net financial wealth
+      values:
+        2021-01-01: 0.00
+      metadata:
+        label: Wealth Tax
+        name: wealth_tax
+        unit: /1
+        max: 0.1
```

Comment:
Just a note that this seems like a good max. Bernie Sanders has the highest proposal I'm aware of, and his top rate is 8%.